src/corelibs/U2Core/src/datatype/primers/PrimerStatistics.h

```diff
@@ -39,9 +39,15 @@ class U2CORE_EXPORT PrimerStatistics : public QObject {
     static double getAnnealingTemperature(const QByteArray &product, const QByteArray &forwardPrimer, const QByteArray &reversePrimer);
 
     static bool validate(const QByteArray &primer);
+    static bool validatePrimerLength(const QByteArray &primer);
```

Comment:
const?

Reply:
Modifiers not allowed on static member functions.